src/yarpdataplayer/include/worker.h

```diff
@@ -130,15 +130,23 @@ class MasterThread : public QObject,  public yarp::os::PeriodicThread
     int                     numThreads;
     double                  timePassed, initTime, virtualTime;
     bool                    stepfromCmd;
+    
+    typedef std::chrono::time_point<std::chrono::steady_clock> Moment;
+    
+    void Initialize();
+    
+    void Tick();
```

Comment:
I personally don't like function names upper case, but it is a personal taste, feel free to ignore it 😉 
Moreover it is not a public interface, then not a big deal.

Reply:
Correct, changed them, thanks